src/backend/Functions/ProcessWorkflowQueue.cs

```diff
@@ -14,17 +15,43 @@ public static class ProcessWorkflowQueue
         [FunctionName("ProcessWorkflowQueue")]
         public static async Task Run([QueueTrigger("apiportworkflowqueue")]WorkflowQueueMessage workflowMessage,
             [Queue("apiportworkflowqueue")]ICollector<WorkflowQueueMessage> workflowMessageQueue,
-            ILogger log)
+            ILogger log,
+            CancellationToken cancelToken)
         {
-            log.LogInformation("Processing message {SubmissionId}, stage {Stage}", workflowMessage.SubmissionId, workflowMessage.Stage);
+            Action<WorkflowQueueMessage> cancelAsync = (msg) =>
+            {
+                log.LogError("Timeout during workflow action execution.SubmissionId: { msg.SubmissionId}, Stage: { msg.Stage}", workflowMessage.SubmissionId, workflowMessage.Stage);
```

Comment:
I would think you should just have `{SubmissionId}` etc in the log string. Is there a reason you include the `msg`?

Reply:
copy paste error...so I removed "msg" now :)